lms/templates/courseware/course_navigation.html

```diff
@@ -29,8 +31,17 @@
 staff_selected = selected(not masquerade or masquerade.role != "student")
 specific_student_selected = selected(not staff_selected and masquerade.user_name)
 student_selected = selected(not staff_selected and not specific_student_selected and not masquerade_group_id)
+include_proctoring_widget = settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False) and course.enable_proctored_exams
```

Comment:
Is "widget" a standardized term used for our UI components?  I'd rather this be more descriptive to what the component actually is -- such as a menu, dialog, etc.